build.gradle

```diff
@@ -29,7 +29,8 @@ formatSource {
 dependencies {
 	compile group: "biz.aQute.bnd", name: "aQute.libg", version: "3.5.0"
 	compile group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.5.0"
-	compile group: "com.liferay", name: "com.liferay.project.templates", version: "3.1.3"
+	compile group: 'com.beust', name: 'jcommander', version: '1.72'
```

Comment:
can you remove this now that we have fixed the project dependency with maven local?